src/Microsoft.Health.Encryption.UnitTests/Microsoft.Health.Encryption.UnitTests.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>$(SupportedFrameworks)</TargetFrameworks>
+    <IsPackable>false</IsPackable>
+    <IsTestProject>true</IsTestProject>
```

Comment:
Is this necessary?

Reply:
No, but when I was creating this initially I was having the hardest time getting VS to recognize it as a test project. After I removed these it would switch to a Library... but now it seems to work